src/main/java/teammates/client/scripts/DataMigrationForSanitizedDataInInstructorAttributes.java

```diff
@@ -0,0 +1,138 @@
+        } else {
+            System.out.println("Sanitization fixing done!");
+        }
+    }
+
+    private boolean previewSanitizedDataForInstructor(InstructorAttributes instructor) {
+        boolean hasSanitizedData = checkInstructorHasSanitizedData(instructor);
+        if (hasSanitizedData) {
+            System.out.println("Checking instructor having email: " + instructor.email);
+            if (isSanitizedString(instructor.displayedName)) {
+                System.out.println("displayedName: " + instructor.displayedName);
+                System.out.println("new displayedName: " + fixSanitization(instructor.displayedName));
+            }
+            if (isSanitizedString(instructor.name)) {
+                System.out.println("name: " + instructor.name);
+                System.out.println("new name: " + fixSanitization(instructor.name));
+            }
+            if (isSanitizedString(instructor.role)) {
+                System.out.println("role: " + instructor.role);
+                System.out.println("new role: " + fixSanitization(instructor.role));
+            }
+            System.out.println();
+        }
+        return hasSanitizedData;
+    }
+
+    private boolean isSanitizedString(String s){
+        if (s == null) return false;
+        if ((s.indexOf('<') >= 0) || (s.indexOf('>') >= 0) || (s.indexOf('\"') >= 0) || 
+            (s.indexOf('/') >= 0) || (s.indexOf('\'') >= 0)) {
+            return false;
+        } else if ((s.indexOf("&lt;") >= 0) || (s.indexOf("&gt;") >= 0) || (s.indexOf("&quot;") >= 0) || 
+                    (s.indexOf("&#x2f;") >= 0) || (s.indexOf("&#39;") >= 0) || (s.indexOf("&amp;") >= 0)) {
+            return true;
+        }
+        return false;
+    }
+    
+    private String fixSanitization(String s) {
+        if (isSanitizedString(s)) {
+            return StringHelper.recoverFromSanitizedText(s);
+        }
+        return s;
+    }
+    
+    private boolean checkInstructorHasSanitizedData(InstructorAttributes instructor) {
+        
+        return isSanitizedString(instructor.role) || isSanitizedString(instructor.displayedName) 
+                || isSanitizedString(instructor.name);
+    }
+    
+    private void fixSanitizationForInstructor(InstructorAttributes instructor) {
+        instructor.displayedName = fixSanitization(instructor.displayedName);
+        instructor.name = fixSanitization(instructor.name);
+        instructor.role = fixSanitization(instructor.role);
+    }
+    
+    private void fixSanitizedDataForInstructor(InstructorAttributes instructor) {
+        try {
+            boolean hasSanitizedData = checkInstructorHasSanitizedData(instructor);
+            if (hasSanitizedData) {
+                fixSanitizationForInstructor(instructor);
+                updateInstructor(instructor);
+            }
+        } catch (InvalidParametersException e) {
+            Utils.getLogger().log(Level.INFO, "Student " + instructor.email + " invalid!");
```

Comment:
This should be "Instructor" instead of "Student". Anyway, let's not catch the exception and let the script fail if there are any errors